packages/desktop-client/src/components/mobile/budget/BudgetTable.jsx

```diff
@@ -70,7 +70,10 @@ function ToBudget({ toBudget, onClick, show3Cols }) {
       <Button
         type="bare"
         style={{ maxWidth: sidebarColumnWidth }}
-        onClick={onClick}
+        onPointerUp={e => {
+          e.stopPropagation();
+          onClick?.();
+        }}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏who we need this change? Also: do pointer events work reliably on mobile where there is no mouse cursor?

Reply:
This is so that it aligns with the behavior of react aria buttons (whose migration is being worked on in a separate PR) which stops propagation by default.